client/src/templates/Challenges/components/fill-in-the-blanks.tsx

```diff
@@ -69,14 +69,16 @@ function FillInTheBlanks({
         })}
       </div>
       <Spacer size='medium' />
-      {showFeedback && feedback && (
-        <>
-          <PrismFormatted text={feedback} />
-          <Spacer size='medium' />
-        </>
-      )}
-      <div className='text-center'>
-        {showWrong && <span>{t('learn.wrong-answer')}</span>}
+      <div aria-live='polite'>
+        <div className='text-center'>
+          {showWrong && (
+            <>
+              <span>{t('learn.wrong-answer')}</span>
+              <Spacer size='medium' />
+            </>
+          )}
+        </div>
+        {showFeedback && feedback && <PrismFormatted text={feedback} />}
```

Comment:
Why is the order of `learn.wrong-answer` and `feedback` reversed?

Reply:
So the screen reader announces "Wrong answer" first. It will be something like "Wrong answer. Focus on the last word."

Instead of "Focus on the last word. Wrong answer."